doc/api/next_api_changes/removals/30067-OG.rst

```diff
@@ -0,0 +1,23 @@
+``TransformNode.is_bbox``
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+... is removed. Instead check the object using ``isinstance(..., BboxBase)``.
+
+``rcsetup.interactive_bk``, ``rcsetup.non_interactive_bk`` and ``rcsetup.all_backends``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+... are removed and replaced by ``matplotlib.backends.backend_registry.list_builtin``
+with the following arguments
+
+- ``matplotlib.backends.BackendFilter.INTERACTIVE``
+- ``matplotlib.backends.BackendFilter.NON_INTERACTIVE``
+- ``None``
+
+``BboxTransformToMaxOnly``
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+... is removed. If you rely on this, please make a copy of old the code.
```

Comment:
I think you can replace it with a `BboxTransformTo(LockableBbox(bbox, x0=0, y0=0))`

Reply:
Ping?